addons/web/static/src/legacy/scss/ui.scss

```diff
@@ -182,3 +182,7 @@ span.o_force_ltr {
     // have an useless frontend rule.
     font-family: sans-serif;
 }
+
+.o_cursor_pointer {
+    cursor: pointer;
+}
```

Comment:
If this class is not "legacy" specific, would probably better to be defined in the "modern" codebase (somewhere below `src/webclient`) instead of this file.

Reply:
Yes. @Brieuc-brd , let's move it to `webclient.scss` for now.
As usual, add a tiny note saying that this class should be generated using BTS5 utility API once migrated.